src/pages/docs/chat/getting-started/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Get started with Chat"
+meta_description: "Get started with Ably Chat in your language or framework of choice. Learn how to send and receive messages, track online presence, fetch message history, implement typing indicators, among other features."
+meta_keywords: "Chat, Ably Chat SDKs, realtime messaging, send and receive messages, getting started guides, realtime communication, Ably tutorial, message history, presence and occupancy APIs, Ably CLI Chat"
+redirect_from:
+  - /docs/chat/setup
+---
+
+Get started with Ably Chat by choosing your language or framework.
+
+You'll learn the basics, such as how to connect to Ably, send and receive messages, and manage the status of users with presence. You'll also be introduced to the Ably CLI and your Ably dashboard to interact with, and manage your apps.
```

Comment:
I think we should tailor this to the Chat version as you do more things, and those specific to Chat.